### PR TITLE
datamodels: align attestation result with ar4si + verifier claims

### PR DESCRIPTION
**Pull request by setrofim**
- Align attestation result definitions with
  https://datatracker.ietf.org/doc/draft-ietf-rats-ar4si/
- Add a Veraison extension for verfier-added claims.

### CODE REVIEW

File: datamodels/attestation-results/grammar/json-labels.cddl [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-ar.status-failure = "failure"
-ar.status-success = "success"
-ar.status-unknown = "unknown"
+ar.trust-tier-none = "none"
+ar.trust-tier-affirming = "affirming"
+ar.trust-tier-warning = "warning"
+ar.trust-tier-contraindicated = "contraindicated"
 
 ; attestation-results
-ar.status = "status"
```
Comment (thomas-fossati) [inline datamodels/attestation-results/grammar/json-labels.cddl:7]: if this label is removed the field becomes anonymous.

Comment (setrofim) [reply datamodels/attestation-results/grammar/json-labels.cddl:7]: argh, yes, that's my bad.

Comment (thomas-fossati) [reply datamodels/attestation-results/grammar/json-labels.cddl:7]: awesome, thanks!

**Comment by thomas-fossati** [comment]
I have an (untested but quite solid) suspicion that the examples are now broken?

**Comment by yogeshbdeshpande** [comment]
CI build is failing ?

**Review by yogeshbdeshpande — APPROVED** [review]
In general LGTM! Perhaps one thing, we should do is that we should have a naming convention :like https://github.com/ietf-rats/ietf-corim-cddl/blob/2ee3299fd9c1c328a9db9f9f92034fe449218f2f/TYPOGRAPHY.md

This way it becomes very clear!
Good for now!

**Pull request merged**